lib/stripe/payment_intent_validator.rb

```diff
@@ -14,6 +16,16 @@ def call(payment_intent_id, stripe_account_id)
 
     private
 
+    def payment_intent_id(payment)
+      payment.response_code
+    end
+
+    def stripe_account_id(payment)
```

Comment:
IMO this class is crying out to have a `@payment` to be set from an initializer.